docs/topics/cache.txt

```diff
@@ -839,9 +839,21 @@ If the object doesn't exist in the cache, ``cache.get()`` returns ``None``::
     >>> cache.get('my_key')
     None
 
-We advise against storing the literal value ``None`` in the cache, because you
-won't be able to distinguish between your stored ``None`` value and a cache
-miss signified by a return value of ``None``.
+If you need to determine whether the object exists in the cache and you have
+stored a literal value ``None``, use a sentinel object as the default::
+
+    >>> sentinel = object()
+    >>> cache.get('my_key', sentinel) is sentinel
+    False
+    >>> # Wait 30 seconds for 'my_key' to expire...
+    >>> cache.get('my_key', sentinel) is sentinel
+    True
+
+.. versionchanged:: 3.2
+
+    In older versions we advised against storing the literal value ``None`` in
+    the cache, because you wouldn't be able to distinguish between your stored
+    ``None`` value and a cache-miss signified by a return value of ``None``.
```

Comment:
It looks that we missed to update this docs in Django 3.0, see [comment](https://code.djangoproject.com/ticket/30181#comment:7) and 0104b5a41704430aaa1067da2281a86a83c8543a. I will chop `versionchanged` annotation.

Reply:
I moved this change to a separate commit.